DRAFT_homogeneous_impact_areas.Rmd

```diff
@@ -0,0 +1,426 @@
+Note that there is still considerable human activity also there in the form of outdoor recreation and even forestry.
+
+#### Aggregate
+
+The resolution in this map is more than we need, and the size of the data implies that the next step, the vectorization, would take too long.
+We therefore aggregate, ar reduce the resolution.
+
+```{r}
+dim <- st_dimensions(infra)
+paste("Resolution is", dim$x$delta, "by", dim$x$delta, "meters")
+```
+
+```{r, warning=F, message=F}
+infra_trd_reclassed_agg <- st_warp(infra_trd_reclassed, cellsize = c(1000, 1000), 
+                                   crs = st_crs(infra_trd_reclassed), 
+                                   use_gdal = TRUE,
+                                   method = "average")
+```
+
+```{r}
+dim <- st_dimensions(infra_trd_reclassed_agg)
+paste("Resolution is", dim$x$delta, "by", dim$x$delta, "meters")
+```
+
+Each cell is now the average of the aggregated cells, and hence the value is continuous again.
+Let's make it discrete.
+
+```{r}
+names(infra_trd_reclassed_agg) <- "infrastructureIndex"
+infra_trd_reclassed_agg <-  infra_trd_reclassed_agg %>%
+  mutate(infrastructureIndex = case_when(
+    infrastructureIndex < 1 ~ 0,
+    infrastructureIndex < 6 ~ 1,
+    infrastructureIndex < 12 ~ 2,
+    infrastructureIndex >= 12 ~ 3
+  ))
+```
+
+
+```{r, fig.cap="Infrastrcture indeks over Trondheim on a four-step discrete scale, aggregated to 1x1 km resolution using the mean function."}
+tm_shape(infra_trd_reclassed_agg)+
+  tm_raster(title="Infrastructure Index",
+            style="cat")+
+  tm_layout(legend.outside = T)+
+  tm_shape(hw_utm)+
+  tm_lines(col="white")
+```
+
+This resolution is more that good enough for our purpose here.
+I see that the map extends a bit into the fjord. We want to cut away these areas. We can use a map of the outline of Norway to do this.
+
+```{r}
+outline <- st_read("data/outlineOfNorway_EPSG25833.shp")
+```
+
+The CRS needs to be identical 
+```{r}
+infra_trd_reclassed_agg <- st_transform(infra_trd_reclassed_agg, 25833)
+```
+
+
+```{r}
+infra_trd_reclassed_agg_terrestrial <- st_crop(infra_trd_reclassed_agg, outline)
+```
+
+```{r, fig.cap="Infrastrcture indeks over Trondheim on a four-step discrete scale, aggregated to 1x1 km resolution using the mean function."}
+tm_shape(infra_trd_reclassed_agg_terrestrial)+
+  tm_raster(title="Infrastructure Index",
+            style="cat")+
+  tm_layout(legend.outside = T)+
+  tm_shape(hw_utm)+
+  tm_lines(col="white")+
+  tm_shape(outline)+
+  tm_borders(lwd =2,
+             col = "black")
+```
+
+I have treated the raster cells as points when cropping, so that cells where the center or the cell is outside he terrestrial delineation are removed. I think this makes the most sense for unbiased area statistics, but some coastal indicator data could also be excluded because of this. The errors would be smaller with a finer resolution raster, but computation time is a problem.
```

Comment:
"center _of_ the cell", "_the_ terrestrial"

Reply:
fixed